<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="inputMask">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>90</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>380</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Modify mask</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="secs">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>310</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>310</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>secs</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="deleteInputCheckBox">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>140</y>
      <width>251</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Delete input file</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="log">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>551</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="overwriteCheckBox">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>250</y>
      <width>251</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Overwrite output file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="chooseOutputButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>190</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose output folder</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButtton">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>470</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="modifyMask">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>380</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="repeatCheckBox">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>310</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Repeat every</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>90</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Iput file mask</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
